modules/tinymce/src/plugins/image/main/ts/api/Settings.ts

```diff
@@ -52,9 +49,7 @@ export {
   hasImageCaption,
   getImageList,
   hasUploadUrl,
-  hasUploadHandler,
   getUploadUrl,
```

Comment:
A number of other settings need to be removed here, as they are no longer being used by the image plugin:
- getUploadUrl
- getUploadBasePath
- getUploadCredentials
- isAutomaticUploadsEnabled

- [ ] Cleanup additional unused settings

Reply:
Done